doc/developer/common/api/json_api.md

```diff
@@ -5,7 +5,7 @@ The hitobito JSON:API implements the open standard [json:api](https://jsonapi.or
 
 Visit your hitobito's [Swagger UI](/api-docs) for detailed documentation and a sandbox for testing/developing requests. (Reload if you get a blank page on the first visit).
 
-This documentation is about the new JSON API introduced in 2023. Check the [legacy Api documentation](rest_api.md) for the old Api.
+This documentation is about the new JSON API introduced in 2023. Check the [legacy Api documentation](/doc/developer/common/api/rest_api.md) for the old Api.
```

Comment:
RubyMine was still able to resolve this link, and it fixes the broken links like https://github.com/hitobito/hitobito/tree/master/doc/developer/rest_api.md in the `/api` documentation page.